common/mas_knowledge_utils/ontology_query_interface.py

```diff
@@ -17,18 +17,20 @@ class OntologyQueryInterface(object):
     Constructor arguments:
     @param ontology_file -- full URL of an ontology file (of the form file://<absolute-path>)
     @param class_prefix -- class prefix of the items in the given ontology
+    @param verbose -- boolean to enable debug console logging, which is disabled by default.
 
     @author Alex Mitrevski
```

Comment:
@Sushant-Chavan Feel free to add yourself as an author; you've already contributed a lot to the interface.